<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointPropertiesWidget</class>
 <widget class="QWidget" name="PointPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="numberOfPointsTextLabel_3">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Number Of Points</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numberOfPointsLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="pointRadiusTextLabel_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Point Radius in 3D</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="pointRadiusSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="pointRadius2DTextLabel_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Point Radius in 2D</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="pointRadius2DSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QLabel" name="LabelSizeTextLabel_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Label Size</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="labelSizeSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>8</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="showLabelsCheckBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Show Labels</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showDistanceCheckBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Show Distance</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="pointsTreeWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <attribute name="headerCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Label</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Position</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="deletePointButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
